IdentityCore/src/broker_operation/request/MSIDBrokerOperationRemoveAccountRequest.m

```diff
@@ -0,0 +1,81 @@
+//
+//  MSIDBrokerOperationRemoveAccountRequest.m
+//  IdentityCore iOS
+//
+//  Created by JZ on 10/9/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "MSIDBrokerOperationRemoveAccountRequest.h"
+#import "MSIDBrokerOperationRequestFactory.h"
+#import "NSDictionary+MSIDExtensions.h"
+#import "MSIDAccountIdentifier+MSIDJsonSerializable.h"
+
+@implementation MSIDBrokerOperationRemoveAccountRequest
+
++ (void)load
+{
+    [MSIDBrokerOperationRequestFactory registerOperationRequestClass:self operation:self.operation];
+}
+
+#pragma mark - MSIDBrokerOperationRequest
+
++ (NSString *)operation
+{
+    return @"remove_account";
+}
+
+#pragma mark - MSIDJsonSerializable
+
+- (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)error
+{
+    self = [super initWithJSONDictionary:json error:error];
+    
+    if (self)
+        {
+            if (![json msidAssertType:NSDictionary.class ofKey:@"request_parameters" required:YES error:error]) return nil;
```

Comment:
We need to decide if we wanna be consistent with "login" and "refresh" operations, where we will use query parameters for serialization/deserialization. Most probably it is better to use query params here as well, but let's discuss.